src/app/src/main/kotlin/org/imperial/mrc/hint/security/tokens/SignatureConfig.kt

```diff
@@ -0,0 +1,18 @@
+package org.imperial.mrc.hint.security.tokens
+
+import org.pac4j.jwt.config.signature.SignatureConfiguration
+import org.pac4j.jwt.config.signature.RSASignatureConfiguration
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import java.security.KeyPair
+
+@Configuration
+class SignatureConfig
```

Comment:
This is a bit confusing - a class called `SignatureConfig` whose only method returns a class of type `SignatureConfiguration` :confused: 

In general I'm a bit unsure about this pattern - we seem to have a proliferation of classes whose only role is to expose a single Bean (https://github.com/mrc-ide/hint/blob/master/src/app/src/main/kotlin/org/imperial/mrc/hint/security/Pac4jConfig.kt#L20, https://github.com/mrc-ide/hint/blob/master/src/app/src/main/kotlin/org/imperial/mrc/hint/security/Pac4jProfileService.kt)

All these examples are security related, so one option would be to have a single SecurityConfig class that defines multiple Beans. But I notice that here you've defined a Bean without a wrapping class at all - https://github.com/mrc-ide/hint/pull/23/files#diff-c044958f508e9341914a1a4f2cbb9e84R16 That seems like a nicer pattern to me, we may as well take advantage of Kotlin not requiring functions to be class methods. What do you think?

Reply:
It was originally called `SignatureConfigurationConfig` which was even worse! 

OK, that sounds good, I'll herd all those security beans into a single SecurtyConfig file, but have them as beans outside of classes.